### ISSUE TITLE
i think there's a problem in the meaning of this sentence...

### ISSUE THREAD

**Issue by roquint** [unraid/lang-en_US#67]
https://github.com/unraid/lang-en_US/blob/f47ae238d865d135fd587eace6a570ba70008067/Parity%20Check%20Tuning/paritychecktuning.txt#L128

it's written : 
The Resume value must be greater than the Pause one...

i think that's the Pause value is when the disk reach an too high temperature, 
and the Resume value is when the disk temperature is nominal and disk operation can resume
so the Pause value will be greater than the resume value, no ?

**Comment by itimpi** [comment]
correct.  

I will reword this to distinguish between the temperatures and the Pause/Resume values you set.
